### ISSUE TITLE
A problem with  LoadH5ADobs()

### ISSUE THREAD

**Issue by CampingOnSeabed** [satijalab/azimuth#254]
> LoadH5ADobs("E:\\Desktop\\BPCells\\humanbrain/modified_data_withanno_100k_cells.h5ad")
Error in H5Fopen(file, flags = flags, fapl = fapl, native = native) : 
  HDF5. File accessibility. Unable to open file.
